x-pack/plugins/aiops/public/components/log_categorization/log_categorization_page.tsx

```diff
@@ -0,0 +1,333 @@
+        index,
+        selectedField,
+        timeField,
+        earliest,
+        latest,
+        searchQuery,
+        intervalMs
+      );
+
+      setCategories(resp.categories);
+      setSparkLines(resp.sparkLinesPerCategory);
+    } catch (error) {
+      toasts.addError(error, {
+        title: i18n.translate('xpack.aiops.logCategorization.errorLoadingCategories', {
+          defaultMessage: 'Error loading categories',
+        }),
+      });
+    }
+
+    setLoading(false);
+  }, [
+    selectedField,
+    dataView,
+    searchQuery,
+    earliest,
+    latest,
+    runCategorizeRequest,
+    cancelRequest,
+    intervalMs,
+    toasts,
+  ]);
+
+  const onFieldChange = (value: EuiComboBoxOptionOption[] | undefined) => {
+    setSelectedField(value && value.length ? value[0].label : undefined);
+  };
+
+  return (
+    <EuiPageBody data-test-subj="aiopsExplainLogRateSpikesPage" paddingSize="none" panelled={false}>
+      <EuiFlexGroup gutterSize="none">
+        <EuiFlexItem>
+          <EuiPageContentHeader className="aiopsPageHeader">
+            <EuiPageContentHeaderSection>
+              <div className="dataViewTitleHeader">
+                <EuiTitle size="s">
+                  <h2>{dataView.getName()}</h2>
+                </EuiTitle>
+              </div>
+            </EuiPageContentHeaderSection>
+
+            <EuiFlexGroup
+              alignItems="center"
+              justifyContent="flexEnd"
+              gutterSize="s"
+              data-test-subj="aiopsTimeRangeSelectorSection"
+            >
+              {dataView.timeFieldName !== undefined && (
+                <EuiFlexItem grow={false}>
+                  <FullTimeRangeSelector
+                    dataView={dataView}
+                    query={undefined}
+                    disabled={false}
+                    timefilter={timefilter}
+                  />
+                </EuiFlexItem>
+              )}
+              <EuiFlexItem grow={false}>
+                <DatePickerWrapper />
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiPageContentHeader>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+      <EuiSpacer />
+      <EuiFlexGroup gutterSize="none">
+        <EuiFlexItem>
+          <SearchPanel
+            dataView={dataView}
+            searchString={searchString ?? ''}
+            searchQuery={searchQuery}
+            searchQueryLanguage={searchQueryLanguage}
+            setSearchParams={setSearchParams}
+          />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+      <EuiSpacer size="m" />
+      <EuiFlexGroup gutterSize="none">
+        <EuiFlexItem grow={false} css={{ minWidth: '410px' }}>
+          <EuiFormRow
+            label={i18n.translate('xpack.aiops.logCategorization.categoryFieldSelect', {
+              defaultMessage: 'Category field',
+            })}
+          >
+            <EuiComboBox
+              isDisabled={loading === true}
+              options={fields}
```

Comment:
What causes the examples column to be blank for certain fields? Here I picked `machine.os` in the sample data logs:

![image](https://user-images.githubusercontent.com/7405507/190177635-620e737d-b18c-4cd6-af7e-36c3d70ba7a1.png)

Reply:
This is caused by the field name containing a period.
Fixed in https://github.com/elastic/kibana/pull/139005/commits/76994e21f8c009af11e73b7dabf2b0839978dcc0